### PR TITLE
Fixes needed for repoclosure

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.16
* [ ] 1.15
* [ ] 1.14

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ehelms** [comment]
@ekohl Found these during the repoclosure run

File: comps/comps-foreman-rhel7.xml [hunk 1]
```diff
@@ -293,6 +294,7 @@
       <packagereq type="default">rubygem-clamp-doc</packagereq>
       <packagereq type="default">rubygem-concurrent-ruby-doc</packagereq>
       <packagereq type="default">rubygem-fog-ovirt-doc</packagereq>
+      <packagereq type="default">rubygem-fog-ovirt-doc</packagereq>
```
Comment (mmoll) [inline comps/comps-foreman-rhel7.xml]: that line is doubled?

**Comment by ehelms** [comment]
Good catch @mmoll -- that was a general typo in the comps as well. I fixed everything up.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**